tasks.py

```diff
@@ -24,6 +25,13 @@ def prepare(ctx, version):
     updated.  New entries will end up at the top of the file, under a heading
     for the new version.
     """
+    # Ensure we're on the master branch first
+    git_rev_parse = ctx.run('git rev-parse --abbrev-ref HEAD', hide=True)
+    current_branch = git_rev_parse.stdout.strip()
+    if current_branch != 'rel':
```

Comment:
why `rel` here? shouldn't be `master`?

I mean, it checks if we are in `rel` and if not, it says "hey, you need to be on master"

Other, we want to ensure we are on `rel` not on `master`, right?